src/App.js

```diff
@@ -11,8 +13,15 @@ import Onboarding from './onboarding/Onboarding'
 const PROVIDER = new Web3.providers.WebsocketProvider('ws://localhost:8545')
 const WALLET_PROVIDER = window.web3.currentProvider
 const ENS = '0x409ba3dd291bb5d48d5b4404f5efa207441f6cba'
+const NETWORK = {
+  etherscanBaseUrl: makeEtherscanBaseUrl('rinkeby'),
+  name: 'Rinkeby testnet',
+}
```

Comment:
Ideally we'd have a couple of different objects for each environment and either select one based on the URL or set them via a build parameter.

Reply:
Since the wrapper will also be used for running apps locally (with `aragon-dev-cli/aragon run`), we also need to support networks with no Etherscan

Reply:
Actually, I saw the `AddressLink` component and it looks like you already had that in mind, but just wanted to doubley-check since we might get some annoying errors if we're not careful. 🙈

Reply:
Good point, I'll change `makeEtherscanBaseUrl()` to only return a string if one of `''` (same as `'main'`), `'main'`, `'rinkeby'`, `'kovan'`, or `'ropsten'` (any others I'm missing?) is given and add an `<EtherscanLink />` component to not display if the url's not available.

Reply:
See b9cb79d.